src/redteam/types.ts

```diff
@@ -153,3 +153,48 @@ export const redteamConfigSchema = z
   });
 
 export type RedteamConfig = z.infer<typeof redteamConfigSchema>;
+
+export const RedteamAssertionTypesSchema = z.enum([
```

Comment:
can you build this dynamically from ALL_PLUGINS or subCategoryDescriptions?

Reply:
I tried something like this
```
export const RedteamAssertionTypesSchema = z.enum(
  REDTEAM_ALL_PLUGINS.map(plugin => `promptfoo:redteam:${plugin}` as const)
);
```
but this zod stuff is too smart for it's own good, it's scared that the enum will have 0 values so had to
```
export const RedteamAssertionTypesSchema = z.enum([
  'promptfoo:redteam:dummy' as const,
  ...REDTEAM_ALL_PLUGINS.map((plugin) => `promptfoo:redteam:${plugin}` as const),
]);
```
stupid but at least we won't have to update this when adding new plugins now